src/commands.ts

```diff
@@ -197,6 +197,44 @@ export const createBlock = (
   }
 }
 
+export const insertGeneralFootnote = (
+  tableNode: ManuscriptNode,
+  position: number,
+  state: ManuscriptEditorState,
+  dispatch?: Dispatch
+) => {
+  const tableElementFooter = findChildrenByType(
```

Comment:
In the previous step, you did that. Is there a way to reconstruct things so you end up doing this once?

Reply:
done